ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CommonStorageBackend.scala

```diff
@@ -111,17 +118,25 @@ private[backend] trait CommonStorageBackend[DB_BATCH] extends StorageBackend[DB_
           |
           |""".stripMargin
       )
-    )(rs =>
-      StorageBackend.LedgerEnd(
+    )(rs => {
+      val rawLedgerEnd = rs.getString(1) // SQL NULL is mapped to JVM null
+      val rawEventSeqId = rs.getLong(2) // SQL NULL is mapped to JVM 0
+      val rawEventSeqIdIsNull = rs.wasNull()
```

Comment:
nice catch, previous code was not resilient for having null event-seq-ids (although accidentally it was still correct :) )

Reply:
Addressed by rewriting the code with anorm